src/sentry/api/endpoints/organization_user_reports.py

```diff
@@ -31,17 +31,20 @@ def get(self, request, organization):
             return Response([])
 
         queryset = UserReport.objects.filter(
-            project_id__in=filter_params["project_id"], group__isnull=False
-        ).select_related("group")
+            project_id__in=filter_params["project_id"], group_id__isnull=False
+        )
         if "environment" in filter_params:
-            queryset = queryset.filter(environment__name__in=filter_params["environment"])
+            queryset = queryset.filter(
+                environment_id__in=[env.id for env in filter_params["environment_objects"]]
+            )
         if filter_params["start"] and filter_params["end"]:
             queryset = queryset.filter(
                 date_added__range=(filter_params["start"], filter_params["end"])
             )
 
         status = request.GET.get("status", "unresolved")
         if status == "unresolved":
+            # TODO: Figure out cross-db join
```

Comment:
If you end up querying the set of groups here to trim down the user report list, you might as well populate it onto the user report models. In that case, however, you'd have to check for that in the serializer, to avoid querying the groups a second time.